res/css/views/rooms/_RecentlyViewedButton.pcss

```diff
@@ -19,7 +19,7 @@ limitations under the License.
     width: max-content;
     max-width: 240px;
     max-height: 400px;
-    border: 1px solid rgba($primary-content, 0.1);
+    border: 1px solid var(--cpd-color-gray-300);
```

Comment:
This will break custom themes e.g. https://github.com/aaronraimist/element-themes